open-machine-learning-jupyter-book/assignments/ml-fundamentals/build-classification-model.ipynb

```diff
@@ -0,0 +1,486 @@
+{
```

Comment:
nothing appears in build-classification-models.html

Reply:
the content of the file "build-classification-models.md" can also appear.

Reply:
the content of the file"build-classification-models.md" can also appear.

Reply:
The content of the file"build-classification-models.ipynb" can also appear.